Makefile

```diff
@@ -213,3 +213,11 @@ dep-ensure:
 .PHONY: dev-env
 dev-env:
 	@./hack/build-dev-env.sh
+
+.PHONY: live-docs
+live-docs:
+	@docker run --rm -it -p 3000:3000 -v ${PWD}:/docs aledbf/mkdocs:0.1
```

Comment:
```
aledbf/mkdocs
```
What is this? Something custom?

Reply:
This https://gist.github.com/aledbf/ce94537bf262e0ae91a4b0d6838c193a

Reply:
I will create a repo for this image (I don't want to install mkdocs in my machine to test the docs or run a local copy)

Reply:
@ElvinEfendi repository of the mkdocs image https://github.com/aledbf/mkdocs